externs/browser/html5.js

```diff
@@ -2976,7 +2976,10 @@ Document.prototype.msHidden;
  * @see http://www.w3.org/TR/components-intro/
  * @see http://w3c.github.io/webcomponents/spec/custom/#extensions-to-document-interface-to-register
  * @param {string} type
- * @param {{extends: (string|undefined), prototype: (Object|undefined)}} options
+ * @param {{extends: (string|undefined), prototype: (Object|undefined)}=} options
+ * @return {?} a constructor for the new tag. An unknown type is the best we
```

Comment:
A slightly better type is !Function, which says that the return type is a function and we don't know anything else about it. Still can be downcast at the call site, and prevents errors such as this:

http://closure-compiler-debugger.appspot.com/#input0%3D%252F**%2520%2540type%2520%257Bnumber%257D%2520*%252F%250Avar%2520n%2520%253D%2520f()%253B%26input1%26conformanceConfig%26externs%3D%252F**%2520%2540return%2520%257B!Function%257D%2520*%252F%250Afunction%2520f()%2520%257B%257D%26refasterjs-template%26includeDefaultExterns%3D1%26CHECK_SYMBOLS%3D1%26CHECK_TYPES%3D1%26MISSING_PROPERTIES%3D1